types/subcall.go

```diff
@@ -1,15 +1,56 @@
 package types
 
-import "encoding/json"
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+)
 
-type replyOn string
+type replyOn int
```

Comment:
This is probably private because `replyOn` is lower case. I wonder of this should be public as well …

Reply:
I have no input here!